### PR TITLE
Add activity table

### PR DESCRIPTION
**Pull request by antooni**
Resolves L2B-135

### CODE REVIEW

File: packages/frontend/src/pages/scaling-activity/view/ActivityPage.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function ActivityPage(props: ActivityPageProps) {
           tpsDaily={props.tpsDaily}
           tpsWeeklyChange={props.tpsWeeklyChange}
         />
-        <Chart endpoint={props.apiEndpoint} hideControls></Chart>
+        <Chart endpoint={props.apiEndpoint} hideControls={true}></Chart>
```
Comment (krzkaczor) [inline packages/frontend/src/pages/scaling-activity/view/ActivityPage.tsx:39]: is this in styleguide or what?

Comment (antooni) [reply packages/frontend/src/pages/scaling-activity/view/ActivityPage.tsx]: It got there by the mistake, when I was resolving conflicts, changed it already. Thanks @krzkaczor

File: packages/frontend/src/build/ApiActivity.ts [hunk 2]
```diff
@@ -25,11 +25,37 @@ const getMockData = (): ApiActivity => {
...
+          'timestamp',
+          'daily tx count',
+        ],
+        data: optimismData,
+      },
+      dydx: {
+        types: ['timestamp', 'daily tx count'] as [
+          'timestamp',
+          'daily tx count',
+        ],
```
Comment (michalsidzej) [inline packages/frontend/src/build/ApiActivity.ts:56]:
```suggestion
        types: ['timestamp', 'daily tx count'],
```

File: packages/frontend/src/build/ApiActivity.ts [hunk 3]
```diff
@@ -25,11 +25,37 @@ const getMockData = (): ApiActivity => {
...
+        data: arbitrumData,
+      },
+      optimism: {
+        types: ['timestamp', 'daily tx count'] as [
+          'timestamp',
+          'daily tx count',
+        ],
```
Comment (michalsidzej) [inline packages/frontend/src/build/ApiActivity.ts:49]:
```suggestion
        types: ['timestamp', 'daily tx count'],
```

File: packages/frontend/src/build/ApiActivity.ts [hunk 4]
```diff
@@ -25,11 +25,37 @@ const getMockData = (): ApiActivity => {
...
       types: ['timestamp', 'daily tx count'] as ['timestamp', 'daily tx count'],
-      data: data as [UnixTime, number][],
+      data: data,
+    },
+    projects: {
+      arbitrum: {
+        types: ['timestamp', 'daily tx count'] as [
+          'timestamp',
+          'daily tx count',
+        ],
```
Comment (michalsidzej) [inline packages/frontend/src/build/ApiActivity.ts:42]:
```suggestion
        types: ['timestamp', 'daily tx count'],
```

File: packages/frontend/src/build/ApiActivity.ts [hunk 5]
```diff
@@ -25,11 +25,37 @@ const getMockData = (): ApiActivity => {
     data.push([START.add(i, 'days'), 100_000 + i * 10_000])
   }
 
+  const arbitrumData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.6])
+  const optimismData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.3])
+  const dydxData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.1])
+
   return {
     combined: {
       types: ['timestamp', 'daily tx count'] as ['timestamp', 'daily tx count'],
```
Comment (michalsidzej) [inline packages/frontend/src/build/ApiActivity.ts:34]: Are those necessary? AFAIK Typescript is smart enough to figure this out
```suggestion
      types: ['timestamp', 'daily tx count'],
```

File: packages/frontend/src/pages/scaling-activity/props/getActivityView.ts [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+}
+
+export function getActivityViewEntry(
+  project: Layer2,
+  apiActivity: ApiActivity,
+): ActivityViewEntry {
+  const SECONDS_IN_A_DAY = 24 * 60 * 60
+  const transactionsWeeklyCount =
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    apiActivity.projects[project.id.toString()]?.data.at(-1)?.[1] ?? 0
```
Comment (michalsidzej) [inline packages/frontend/src/pages/scaling-activity/props/getActivityView.ts:25]: Should be the sum of all the week, this is the daily count

Comment (antooni) [reply packages/frontend/src/pages/scaling-activity/props/getActivityView.ts:25]: thanks, great catch

File: packages/frontend/src/pages/scaling-activity/props/getActivityView.ts [hunk 7]
```diff
@@ -1,5 +1,43 @@
...
+  }
+}
+
+export function getActivityViewEntry(
+  project: Layer2,
+  apiActivity: ApiActivity,
+): ActivityViewEntry {
+  const SECONDS_IN_A_DAY = 24 * 60 * 60
+  const transactionsWeeklyCount =
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (sz-piotr) [inline packages/frontend/src/pages/scaling-activity/props/getActivityView.ts:24]: Do we need those here?

**Review by sz-piotr — APPROVED** [review]

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**